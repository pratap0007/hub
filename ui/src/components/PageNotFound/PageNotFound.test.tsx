import React from 'react';
import { shallow } from 'enzyme';
import { PageNotFound } from './index';

jest.mock('react-router-dom', () => {
  return {
    useHistory: () => {
      return {
        history: ''
      };
    }
  };
});

describe('PageNotFound Component', () => {
  it('should render PageNotFound component', () => {
    const component = shallow(<PageNotFound />);
    expect(component.debug()).toMatchSnapshot();
  });
});
