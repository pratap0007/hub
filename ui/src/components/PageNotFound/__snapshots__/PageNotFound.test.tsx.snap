// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageNotFound Component should render PageNotFound component 1`] = `
"<EmptyState variant=\\"small\\">
  <EmptyStateIcon icon={[Function: warningIcon]} />
  <Title headingLevel=\\"h5\\" size=\\"lg\\" className=\\"hub-pagenotfound__title\\">
    Page Not Found
  </Title>
  <Button variant=\\"primary\\" onClick={[Function: onClick]}>
    Back Home
  </Button>
</EmptyState>"
`;
